nservicebus/operations/metrics.md

```diff
@@ -71,4 +71,30 @@ These statistics encompass a number of different metrics, including:
 - Number of message processing failures
 - Number of messages successfully processed
 
+### Queue length
 
+This metric tracks estimated number of messages in the input queue of an endpoint.
+
+A _link_ is a communication channel between a sender of the message and its receiver. Each link is uniquely identified by some combination of destination address, message assembly, and the [host identifier](/nservicebus/hosting/override-hostid.md#host-identifier) of the sender. The exact composition of link identifiers depends on the transport properties and type of message being sent.
+
+Each sender maintains a monotonic counter of messages sent over each of its outgoing links and transmits the value of this counter to the receiver in a message header. The receiver tracks the counter value for the last message received over each link. This allows both communicating endpoints to track how many messages were sent and received over each link.
+
+ServiceControl collects these metrics for all links and estimates the length of the input queue for each receiver based on how many messages were sent in total over all incoming links and how many of those messages have already been received.
+
+#### Example
+
+The system consists of two endpoints, Sales and Shipping. The Sales endpoint is scaled out and deployed to two machines, `1` and `2`. Consider the following values reported to ServiceControl:
```

Comment:
>The system consists of two endpoints, Sales and Shipping

If you could add some more information about how Sales and Shipping are communicating, that would help. From the table below, it appears that Sales is sending messages to Shipping (which is why Shipping has messages in its queue). What is confusing is what is causing messages to be in Sales' queue - the table seems to imply that those messages would be coming from Shipping, but why would Shipping be sending messages to Sales?